src/Tinc/Install.hs

```diff
@@ -99,11 +99,12 @@ findReusablePackages ghcInfo cacheDir installPlan = do
   sandboxes <- lookupSandboxes cacheDir
   globalPackages <- listGlobalPackages
   cachedPackages <- fmap concat . forM sandboxes $ \ sandbox -> do
-    packageDb <- findPackageDb sandbox
-    cacheGraph <- readPackageGraph [ghcInfoGlobalPackageDb ghcInfo, packageDb]
+    packageDbPath <- findPackageDb sandbox
+    cacheGraph <- readPackageGraph [ghcInfoGlobalPackageDb ghcInfo, packageDbPath]
     let packages = nubBy ((==) `on` packageName) (installPlan ++ globalPackages)
-        reusable = calculateReusablePackages packages cacheGraph
-    lookupPackages packageDb reusable
+        reusable = calculateReusablePackages packages cacheGraph \\ globalPackages
```

Comment:
So before `calculateReusablePackages` didn't return the `globalPackages`? And now it does?